db/kvdb/kvdb.go

```diff
@@ -49,6 +49,8 @@ type KVDB struct {
 	CurrentIdx   common.Idx
 	CurrentBatch common.BatchNum
 	m            sync.Mutex
+	mutexDelOld  sync.Mutex
+	wg           sync.WaitGroup
```

Comment:
```suggestion
	wgDelOld           sync.WaitGroup
```

Reply:
This wait group is not specific to deletion, it's main idea to wait for all goroutines in kvdb on app shutdown, so in future there could be other gouroutines to wait for.